code/parachain/frame/pablo/src/lib.rs

```diff
@@ -666,11 +667,13 @@ pub mod pallet {
 				.into_iter()
 				.try_collect()
 				.map_err(|_| Error::<T>::StakingPoolConfigError)?;
-			let duration_presets =
-				[(ONE_WEEK, Perbill::from_percent(1)), (ONE_MONTH, Perbill::from_percent(10))]
-					.into_iter()
-					.try_collect()
-					.map_err(|_| Error::<T>::StakingPoolConfigError)?;
+			let duration_presets = [
+				(ONE_WEEK, FixedU64::from_rational(1, 100)),
```

Comment:
Can we try some values > 1 ?

Reply:
yes, it was not clear what change about when I looked at test. so >1 is good test changes really enacted as per task

Reply:
Do we want to enforce a minimum of > 1?

Reply:
Yeah I think that makes sense given it's supposed to be a multiplier and no one would stake to get less

Reply:
https://github.com/ComposableFi/composable/commit/418d5842bbb19e7c772b31e079e099ee5331d519